build/wd_test.bzl

```diff
@@ -199,6 +204,13 @@ _wd_test = rule(
             executable = True,
             cfg = "exec",
         ),
+        "sidecar_num_ports_to_assign": attr.int(),
```

Comment:
@anonrig As discussed in DMs, I improved the name of this attribute. It's now "sidecar_num_ports_to_assign".